src/textual/reactive.py

```diff
@@ -44,6 +44,7 @@ class Reactive(Generic[ReactiveType]):
         repaint (bool, optional): Perform a repaint on change. Defaults to True.
         init (bool, optional): Call watchers on initialize (post mount). Defaults to False.
         always_update (bool, optional): Call watchers even when the new value equals the old value. Defaults to False.
+        no_compute (bool, optional): Don't run compute methods when attribute is changed. Defaults to False.
```

Comment:
Might just be me, but the double negative always takes some to process. Calling it `compute` and defaulting to `True` would make it easier to parse, no?

Reply:
Agree. Double negative was awkward.